shopinvader/models/shopinvader_variant.py

```diff
@@ -312,16 +322,42 @@ def get_value(record, key):
                 else:
                     return record[key]
 
-            ordered = sorted(
-                prods, key=lambda var: [get_value(var, x) for x in order_by]
-            )
+            try:
+                ordered = sorted(
+                    variants, key=lambda var: [get_value(var, x) for x in order_by]
+                )
+            except TypeError as orig_exception:
```

Comment:
@simahawk @qgroulard It seems that the cherry-pick pull this Exception part.

Is it necessary ?